pandas/core/series.py

```diff
@@ -2762,23 +2765,68 @@ def nlargest(self, n=5, keep='first'):
 
         See Also
         --------
-        Series.nsmallest
+        Series.nsmallest: Get the `n` smallest elements.
```

Comment:
I'd add `sort_values` and `head` here, as they're mentioned in the notes. In both docstrings.